### ISSUE TITLE
Some notifications and toasts not displaying

### ISSUE THREAD

**Issue by gaul** [helloworld1/AnyMemo#458]
I cannot see `converting_success_text` when importing CSV.  I believe this is due to `ConvertIntentService` being an `IntentService`.  Reference:

https://stackoverflow.com/questions/21515917/android-service-to-show-toast

**Comment by helloworld1** [comment]
Yes, It won't display in a service. I will remove the toast there and rely on push notifications to indicate the success of converting.
